packages/react-integration/cypress/integration/codeeditor.spec.ts

```diff
@@ -10,7 +10,7 @@ describe('Context Selector Demo Test', () => {
       .first()
       .then(code => expect(code).to.have.text('test'));
     cy.get('#reset-code').click();
-    cy.wait(100);
+    cy.wait(1000);
```

Comment:
Why was this necessary timeout increase? Just out of interest!

Reply:
Tests were failing. This was recently updated to `100` to fix the flakes, but apparently it wasn't enough.